x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/new_index/new_search_index_template.tsx

```diff
@@ -170,16 +179,16 @@ export const NewSearchIndexTemplate: React.FC<Props> = ({
                     }
                   )}
                   isInvalid={formInvalid}
-                  error={formError()}
-                  helpText={i18n.translate(
-                    'xpack.enterpriseSearch.content.newIndex.newSearchIndexTemplate.nameInputHelpText.lineOne',
-                    {
-                      defaultMessage: 'Your index will be named: {indexName}',
-                      values: {
-                        indexName: fullIndexName,
-                      },
-                    }
-                  )}
+                  error={
+                    <EuiText size="xs" style={{ lineBreak: 'anywhere' }}>
```

Comment:
A suggestion: Does using `css` property works? If so, please use that, as we cannot overwrite style, due to specificity. 

example use: https://github.com/elastic/kibana/blob/main/x-pack/plugins/enterprise_search/public/applications/shared/api_key/api_key_panel.tsx#L66

Reply:
@efegurkan it does work! I didn't know `css` existed for this, thanks :)